packages/amplication-data-service-generator/src/server/resource/service/create-service.ts

```diff
@@ -244,46 +238,46 @@ async function createServiceBaseModule({
 
   removeTSClassDeclares(template);
 
-  if (passwordFields.length) {
-    const classDeclaration = getClassDeclarationById(template, serviceBaseId);
-
-    addInjectableDependency(
-      classDeclaration,
-      PASSWORD_SERVICE_MEMBER_ID.name,
-      PASSWORD_SERVICE_ID,
-      "protected"
-    );
-
-    for (const member of classDeclaration.body.body) {
-      if (
-        namedTypes.ClassMethod.check(member) &&
-        namedTypes.Identifier.check(member.key) &&
-        PASSWORD_FIELD_ASYNC_METHODS.has(member.key.name)
-      ) {
-        member.async = true;
-      }
-    }
-    //add the password service
-    addImports(template, [
-      importNames(
-        [PASSWORD_SERVICE_ID],
-        relativeImportPath(
-          moduleBasePath,
-          `${serverDirectories.srcDirectory}/auth/password.service.ts`
-        )
-      ),
-    ]);
-
-    addImports(template, [
-      importNames(
-        [TRANSFORM_STRING_FIELD_UPDATE_INPUT_ID],
-        relativeImportPath(
-          moduleBasePath,
-          `${serverDirectories.srcDirectory}/prisma.util.ts`
-        )
-      ),
-    ]);
-  }
+  // if (passwordFields.length) {
+  //   const classDeclaration = getClassDeclarationById(template, serviceBaseId);
+
+  //   addInjectableDependency(
+  //     classDeclaration,
+  //     PASSWORD_SERVICE_MEMBER_ID.name,
+  //     PASSWORD_SERVICE_ID,
+  //     "protected"
+  //   );
+
+  //   for (const member of classDeclaration.body.body) {
+  //     if (
+  //       namedTypes.ClassMethod.check(member) &&
+  //       namedTypes.Identifier.check(member.key) &&
+  //       PASSWORD_FIELD_ASYNC_METHODS.has(member.key.name)
+  //     ) {
+  //       member.async = true;
+  //     }
+  //   }
+  //   //add the password service
+  //   addImports(template, [
+  //     importNames(
+  //       [PASSWORD_SERVICE_ID],
+  //       relativeImportPath(
+  //         moduleBasePath,
+  //         `${serverDirectories.srcDirectory}/auth/password.service.ts`
+  //       )
+  //     ),
+  //   ]);
+
+  //   addImports(template, [
+  //     importNames(
+  //       [TRANSFORM_STRING_FIELD_UPDATE_INPUT_ID],
+  //       relativeImportPath(
+  //         moduleBasePath,
+  //         `${serverDirectories.srcDirectory}/prisma.util.ts`
+  //       )
+  //     ),
+  //   ]);
+  // }
```

Comment:
should this be deleted? or is it still needed?

Reply:
deleted @yuval-hazaz